app/controllers/main/routes.py

```diff
@@ -2,18 +2,33 @@
 from app.models.user import User
 from app.models.eventParticipant import EventParticipant
 from app.models.program import Program
-from app.models.user import User
-from app.models.eventParticipant import EventParticipant
 from app.models.interest import Interest
 from app.models.term import Term
-from app.models.eventParticipant import EventParticipant
 from app.controllers.main import main_bp
 from app.logic.events import getUpcomingEventsForUser
 from app.logic.users import addRemoveInterest
 from app.logic.participants import userRsvpForEvent, unattendedRequiredEvents
+from app.logic.transcript import *
 from app.logic.events import groupEventsByCategory
 from datetime import datetime
 
+@main_bp.route('/')
+def home():
+    print(f"{g.current_user.username}: {g.current_user.firstName} {g.current_user.lastName}")
+    return render_template('main/home.html', title="Welcome to CELTS!")
```

Comment:
Not sure how this old route got added back in. I'm guessing because of merge conflicts, but it makes me worried about other merge conflict resolutions.